tests/unit/corruption.cpp

```diff
@@ -17,8 +17,8 @@ BOOST_AUTO_TEST_CASE(test_header_corruption) {
 	/* corrupt record's header in blob and check that record isn't considered as corrupted (since data is correct)
```

Comment:
I'm not sure. Do me limit line length in commit message? Some commit messages is too wide (out of git recommended style)